src/get.rs

```diff
@@ -42,95 +42,158 @@ pub(crate) async fn get<Data: Send + Sync + 'static>(
     crate::utils::shutdown_check(&fetcher.shutdown)?;
 
     let shutdown = fetcher.shutdown.clone();
-    let FetchLocation { mut file, dest } = file;
-
-    let request = request.body(()).expect("failed to build request");
+    let FetchLocation { file, dest } = file;
 
     let main = async move {
         let _token = match shutdown.delay_shutdown_token() {
             Ok(token) => token,
             Err(_) => return Err(Error::Canceled),
         };
 
-        let req = async {
-            fetcher
-                .client
-                .send_async(request)
-                .await
-                .map_err(Error::from)
-        };
-
-        let initial_response = crate::utils::timed_interrupt(Duration::from_secs(10), req).await?;
-
-        if initial_response.status() == StatusCode::NOT_MODIFIED {
-            return Ok::<_, crate::Error>((dest, file));
+        match &fetcher.client {
+            Client::Isahc(client) => {
```

Comment:
isahc shouldn't be compiled or linked when reqwest is enabled

Reply:
If both features are enabled, both should still be available right?

Reply:
One or the other, but never both at the same time.